### PR TITLE
Add epilog script (HPC-8354)

### PR DESCRIPTION
**Pull request by itkovian**
- clean up leftover shared memory ipcs zone

### CODE REVIEW

File: files/epilog.sh [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    exit 0
+fi
+
+
+${LOGGER} -p local0.alert "********  starting $0 for job $SLURM_JOB_ID"
+
+#
+# Only run for valid VSC user IDs
+#
+if [ $SLURM_UID -lt 2500000 ] ; then
```
Comment (wdpypere) [inline files/epilog.sh:58]: what is this supposed to catch? and if the docs are right, should it nog be higher than a certain number as well? like 1000 or something?

File: files/epilog.sh [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    exit 0
+fi
+
+user_ipcs_cleanup
+
+
+${LOGGER} -p local0.alert "********  finished $0 for job $SLURM_JOB_ID"
```
Comment (wdpypere) [inline files/epilog.sh:65]: if it exits above on UUID it only prints "start" and never the the "finished", is that intentional?

File: slurm-prepilogue.spec [hunk 3]
```diff
@@ -54,12 +55,15 @@ rm -rf %{buildroot}
 /usr/libexec/slurm/prolog/checkpaths.sh
 /usr/libexec/slurm/prolog/checkpaths_stat.sh
 /usr/libexec/slurm/prolog/functions.sh
+/usr/libexec/slurm/prolog/epilog.sh
```
Comment (wdpypere) [inline slurm-prepilogue.spec:58]: you installed it in the epilog dir on line 45, so replace prolog by epilog here

File: files/epilog.sh (hunk 1 above)
Comment (itkovian) [reply files/epilog.sh:58]: VSC user IDs start at 2510001. Not sure what you mean.

File: files/epilog.sh (hunk 2 above)
Comment (itkovian) [reply files/epilog.sh:65]: Right.

File: slurm-prepilogue.spec (hunk 3 above)
Comment (itkovian) [reply slurm-prepilogue.spec:58]: Nice catch, thx.

**Pull request merged**